src/main/java/seedu/address/logic/commands/AddMeetingCommand.java

```diff
@@ -0,0 +1,103 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Meeting;
+import seedu.address.model.person.Person;
+
+/**
+ * Changes the meeting of an existing person in the address book.
+ */
+public class AddMeetingCommand extends Command {
+
+    public static final String COMMAND_WORD = "mtg";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Adds a meeting to the person identified by the contact name.\n"
+            + "mtg [CONTACT_NAME] m/[MTG_DESC] t/[TIMING]\n"
+            + "Example: " + COMMAND_WORD + " Alex Tan "
+            + "m/Interview t/23-03-2024 1600-1700";
+
+    public static final String MESSAGE_ADD_MEETING_SUCCESS = "Added the meeting with %1$s. The meeting details "
+            + "are as follows:\n" + "%2$s\n";
+    public static final String MESSAGE_DELETE_MEETING_SUCCESS = "Removed the meeting with %1$s.";
+
+    public static final String MESSAGE_PERSON_NOT_FOUND = "Oops, %1$s's contact does not exist. Unable to add "
+            + "meeting.";
+    public static final String MESSAGE_EMPTY_NAME = "Oops, please state the name of the contact.";
```

Comment:
It's good that you used different messages based on the different expected outcomes